src/domain/accounts/entities/schemas/create-account.dto.schema.spec.ts

```diff
@@ -68,4 +75,8 @@ describe('CreateAccountDtoSchema', () => {
       );
     });
   });
+
+  describe('name', () => {
+    // TODO: Add tests for name validation
```

Comment:
👀

Reply:
Added in [2069930](https://github.com/safe-global/safe-client-gateway/pull/2081/commits/2069930ce18518a38b763ad69b1a7585e6f2016e)